sections/next-story.liquid

```diff
@@ -0,0 +1,22 @@
+<div class="next-story">
+    <p class="next-story__title">LOREM IPSUM DOLOR SUM</p>
+    <p class="next-story__body">
+            Vestibulum rutrum quam vitae fringilla tincidunt. Suspendisse nec tortor urna.
+            Ut laoreet sodales nisi, quis iaculis nulla iaculis vitae. Donec sagittis faucibus 
+            lacus eget blandit. Mauris vitae ultricies metus, at condimentum nulla. Donec quis ornare lacus. 
+            Etiam gravida mollis tortor quis porttitor. </p>
+</div>
+<img class="img-in-next-story" src="{{section.settings.img_next_story | img_url: '800x'}}">
+
+{% schema %}
+{
+  "name": "img-next-story",
```

Comment:
you could write the Schema's name like `Next Story Image`
remember that this is going to be uses by the final user and could be better read `Next Story Image` than `img-next-story` from the admin